2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 1 Line 9_
```markdown

# Feature importance
During this workshop we will explore three different ways to estimate the importance of our input features on the model outputs: impurity feature importance, permutation feature importance and shap feature importance.

## Setting up our (random forest) model
To begin estimating feature importance we need a model. We will use the same model as in the "understanding random forest" workshop. Surprise, surprise, it's a random forest model. The code blocks below (1) loads the data, (2) splits our data into training and testing datasets, (3) trains our random forest model and (4) provides a simple evaluation of the model performance.

Note that this code is near identical to the "understanding random forest" workshop, If you have any problems understanding what is happening, please take a look there.
```
I really like that you really get to the point here and refer to the previous workshop (maybe even add a link to it, or mention the filename in the repo) , but I think it still would be nice to add a quick description of the data, especially of the features involved, as we will discuss their importance later in detail and people should not be going back and forth all the time.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
Done